packages/coil-extension/static/rules.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "id": 1,
+    "priority": 1,
+    "action": {
+      "type": "modifyHeaders",
+      "requestHeaders": [
+        { "header": "accept-payment", "operation": "set", "value": "webmon/*;q=0.8" }
```

Comment:
Ideally this `set` would be an `append` but apparently there's an [open bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1117475) that prevents that

Reply:
Will it matter much in practice do you think ? 
There will be races for which extension gets to set it in the case of more than one enabled ?

Reply:
In the start it won't matter.

Eventually it could be a race condition. Ideally you'd want an adblock extension to be able to say "no ads" as well as what we have here. I'd think by the time those extensions implement and there's a supporting ux the bug would be fixed.